testapp/tests/test_generic.py

```diff
@@ -452,7 +452,7 @@ def test_run_async(self, mp, tp, job_prefix=''):
                             'name': f'{job_prefix}tmp_somecommand_1',
                             'schedule': '@at 2023-02-07T00:00:05+00:00',
                             'executor_config': {
-                                'command': 'python ./manage.py somecommand arg1 --kwarg value --enable'
+                                'command': 'python ./manage.py run_dkron_async_command somecommand eyJhcmdzIjogWyJhcmcxIl0sICJrd2FyZ3MiOiB7Imt3YXJnIjogInZhbHVlIiwgImVuYWJsZSI6IHRydWV9fQ=='
```

Comment:
I believe this text bit does not belong here.
```suggestion
                                'command': 'python ./manage.py run_dkron_async_command eyJhcmdzIjogWyJhcmcxIl0sICJrd2FyZ3MiOiB7Imt3YXJnIjogInZhbHVlIiwgImVuYWJsZSI6IHRydWV9fQ=='
```

Reply:
It does, target `command` is the first argument. base64 of args and kwargs is the second one.

Btw, this is a unit test, it would fail if it wasn't aligned with the code 😄

Reply:
Good point about the tests, yes. I thought the target command here was `run_dkron_async_command` and the args was the b64 payload. Thanks!